modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/exportimport/content/processor/SegmentsEntryExportImportContentProcessor.java

```diff
@@ -120,6 +132,91 @@ private String _replaceExportCriteriaReferences(
 		return CriteriaSerializer.serialize(criteria);
 	}
 
+	private String _replaceExportExpandoReferences(String content) {
+		Matcher matcher = _exportCustomFieldPattern.matcher(content);
+
+		StringBuffer sb = null;
+
+		while (matcher.find()) {
+			if (sb == null) {
+				sb = new StringBuffer(content.length());
+			}
+
+			String columnId = matcher.group(1);
+
+			ExpandoColumn expandoColumn =
+				_expandoColumnLocalService.fetchExpandoColumn(
+					GetterUtil.getLong(columnId));
+
+			if (expandoColumn == null) {
+				continue;
+			}
+
+			StagedExpandoColumn stagedExpandoColumn = ModelAdapterUtil.adapt(
+				expandoColumn, ExpandoColumn.class, StagedExpandoColumn.class);
+
+			String uuid = stagedExpandoColumn.getUuid();
+
+			matcher.appendReplacement(
+				sb,
+				Matcher.quoteReplacement(
+					"customField\\/_" + uuid.replaceAll(" ", "_")));
+		}
+
+		if (sb != null) {
+			matcher.appendTail(sb);
+
+			content = sb.toString();
```

Comment:
What do you think about return directly sb.toString()?

Reply:
that could work, I'm just a fan of single returns